server/terraform/README.md

```diff
@@ -167,6 +167,7 @@ which are in the list below. The project renaming is still in progress:
 | who-mh-prod-in-dev | Replicates permissions and setup of who-mh-prod project |
 | who-mh-prod        | WHO Production infrastructure, Privacy Policy compliant |
 | who-mh-staging     | CI server updated from GitHub repo                      |
+| who-mh-v2-dev1     | Development (unstable) environment for v2 backend       |
```

Comment:
@rjhuijsman 
I was wondering about doing `who-mh2-dev1` instead? I think that'll be less to type and easier to read than an explicit `-v2-`.... which should only be relevant very briefly. You want the developed eye drawn to the `-dev1` which should be the only thing that they need to know.

Reply:
I like that. Done. Note, I still need to delete the `who-mh-v2-dev1` project, I'll leave that until we're 100% sure we don't want it anymore.